test/README.md

```diff
@@ -0,0 +1,14 @@
+# RT-tests smoke tests 
+
+These are smoke tests that check if the main functionalities of each program is working.
+
+These tests are packaging agnostic, so doesn't matter how did you installed the `rt-tests` package.
+
+
+## Running tests
+
```

Comment:
Please explain and provide the commands for the required snap aliasing (or add commands to top-level readme and link to it).